packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -747,14 +747,9 @@ export class ContainerRuntime
 
 		const initializeEntryPoint =
 			params.initializeEntryPoint ??
```

Comment:
Shouldn't `initializeEntryPoint` become required in `loadRuntime`? Sorry, I forget if there was a reason it can't be yet.